pkg/kubelet/lifecycle/handlers.go

```diff
@@ -55,9 +58,16 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
 		var buffer bytes.Buffer
 		output := ioutils.WriteCloserWrapper(&buffer)
 		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false)
+		if err != nil {
+			glog.V(1).Infof("Lifecycle-Probe for Container %q in Pod %q failed - %q", container.Name, pod.Name, buffer.String())
```

Comment:
nit: use the `pkg/kubelet/util/format` util to print the pod information:
s/pod.Name/format.Pod(pod)

nit2: how about replacing "Probe" with "hook" and print the handler or command, so that it's clear which hook failed. 

Same for the http handler below.